tests/jballerina-integration-test/src/test/java/org/ballerinalang/test/configurables/ConfigurableTest.java

```diff
@@ -176,6 +185,61 @@ public void testNoModuleInTOML() throws BallerinaTestException {
         errorLeecher.waitForText(5000);
     }
 
+    @Test
+    public void testUnsupportedRecordField() throws BallerinaTestException {
+        Path projectPath = Paths.get(negativeTestFileLocation, "InvalidRecordField").toAbsolutePath();
+        LogLeecher errorLeecher =
+                new LogLeecher("Configurable feature is yet to be supported for field type " +
+                        "'string[][]' in variable 'testUser' of record 'main:AuthInfo'", ERROR);
+        bMainInstance.runMain("run", new String[]{"main"}, null, new String[]{},
+                new LogLeecher[]{errorLeecher}, projectPath.toString());
+        errorLeecher.waitForText(5000);
+    }
+
+    @Test
+    public void testInvalidAdditionalRecordField() throws BallerinaTestException {
+        Path projectPath = Paths.get(negativeTestFileLocation, "AdditionalField").toAbsolutePath();
+        LogLeecher errorLeecher =
+                new LogLeecher("Additional field 'scopes' provided for configurable variable 'testUser' of record " +
+                        "'main:AuthInfo' is not supported", ERROR);
```

Comment:
Grammatically it should be `and it is not supported`, but let's remove that part since it sounds excessive.